packages/EasyCodingStandard/config/common/spaces.yml

```diff
@@ -59,3 +59,10 @@ services:
     PhpCsFixer\Fixer\FunctionNotation\MethodArgumentSpaceFixer: ~
 
     PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\LanguageConstructSpacingSniff: ~
+
+    SlevomatCodingStandard\Sniffs\Classes\TraitUseDeclarationSniff:
+    SlevomatCodingStandard\Sniffs\Classes\TraitUseSpacingSniff:
+        linesCountAfterLastUse: 1
+        linesCountAfterLastUseWhenLastInClass: 0
+        linesCountBeforeFirstUse: 0
+        linesCountBetweenUses: 0
```

Comment:
I think this is alraedy covered by rules from PHP CS Fixer.

Could you try them and prove that this is needed?

Reply:
Okay, gonna search and revert if so.

Reply:
Just create `demo/Traits.php` with tset code and run current set:

```
packages/EasyCodingStandard/bin/ecs check demo 
```

Reply:
All right then, thanks for verification :+1: